integration-cli/docker_api_images_test.go

```diff
@@ -62,7 +62,7 @@ func (s *DockerSuite) TestApiImagesSaveAndLoad(c *check.C) {
 
 	defer body.Close()
 
-	dockerCmd(c, "rmi", id)
+	dockerCmd(c, "rmi", "-f", id)
```

Comment:
I'd argue that this requires `--force` because we are deleting the image by ID but it is currently tagged as `saveandload:latest`. Not forcing it results in an error saying that the image is referenced in one or more repositories and that it must be force removed (thus removing any repository references).